Content.Server/GameObjects/Components/Damage/DamageableComponent.cs

```diff
@@ -3,23 +3,34 @@
 using System.Linq;
 using Content.Server.Interfaces;
 using Content.Server.Interfaces.GameObjects;
+using Content.Shared.Audio;
 using Content.Shared.GameObjects;
+using Robust.Server.GameObjects.EntitySystems;
+using Robust.Shared.Audio;
 using Robust.Shared.GameObjects;
+using Robust.Shared.GameObjects.Systems;
 using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Interfaces.Random;
+using Robust.Shared.IoC;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
 using Robust.Shared.Serialization;
 using Robust.Shared.ViewVariables;
 
 namespace Content.Server.GameObjects
 {
-    //TODO: add support for component add/remove
-
     /// <summary>
     /// A component that handles receiving damage and healing,
     /// as well as informing other components of it.
     /// </summary>
     [RegisterComponent]
     public class DamageableComponent : SharedDamageableComponent, IDamageableComponent
     {
+        #pragma warning disable 649
+        [Dependency] private readonly IPrototypeManager _prototypeManager;
+        [Dependency] private readonly IRobustRandom _random;
+        #pragma warning restore 649
```

Comment:
These should be resolved as the sound is played because many things are damageable (e.g. walls) but very few are playing sounds at any one time.

Reply:
Also pragma no longer needed for future reference.